README.md

```diff
@@ -140,38 +140,53 @@ the table COPY, cumulative timing for the CREATE INDEX commands), and then
-18:26:36 77615 INFO  STEP 2: restore the pre-data section to the target database
-18:26:36 77615 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54311 dbname=plop' /tmp/pgcopydb/schema/schema.dump
-18:26:36 77615 INFO  STEP 3: copy data from source to target in sub-processes
-18:26:36 77615 INFO  STEP 4: create indexes and constraints in parallel
-18:26:36 77615 INFO  STEP 5: vacuum analyze each table
-18:26:36 77615 INFO  Listing ordinary tables in "port=54311 host=localhost dbname=pgloader"
-18:26:36 77615 INFO  Fetched information for 56 tables
-...
-18:26:37 77615 INFO  STEP 6: restore the post-data section to the target database
-18:26:37 77615 INFO   /Applications/Postgres.app/Contents/Versions/12/bin/pg_restore --dbname 'port=54311 dbname=plop' --use-list /tmp/pgcopydb/schema/post.list /tmp/pgcopydb/schema/schema.dump
-
-  OID |   Schema |            Name | copy duration | indexes | create index duration
-------+----------+-----------------+---------------+---------+----------------------
-17085 |      csv |           track |          62ms |       1 |                  24ms
-  ...
-  ...
-
-                                          Step   Connection    Duration   Concurrency
- ---------------------------------------------   ----------  ----------  ------------
-                                   Dump Schema       source       884ms             1
-                                Prepare Schema       target       405ms             1
- COPY, INDEX, CONSTRAINTS, VACUUM (wall clock)         both       1s281         8 + 2
-                             COPY (cumulative)         both       2s040             8
-                     CREATE INDEX (cumulative)       target       381ms             2
-                               Finalize Schema       target        29ms             1
- ---------------------------------------------   ----------  ----------  ------------
-                     Total Wall Clock Duration         both       2s639         8 + 2
- ---------------------------------------------   ----------  ----------  ------------
+07:54:31.646 40543 INFO   [SOURCE] Copying database from "postgres://user@source.host.dev/dbname?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
+07:54:31.646 40543 INFO   [TARGET] Copying database into "postgres://role@target.host.dev/dbname?keepalives=1&keepalives_idle=10&keepalives_interval=10&keepalives_count=60"
+07:54:31.711 40543 INFO   Using work dir "/tmp/pgcopydb"
+07:54:31.719 40543 INFO   Exported snapshot "00000003-000000D1-1" from the source database
+07:54:31.722 40554 INFO   STEP 1: fetch source database tables, indexes, and sequences
+07:54:32.046 40554 INFO   Fetched information for 1 tables (including 0 tables split in 0 partitions total), with an estimated total of 18 quintillion tuples and 8192 B on-disk
```

Comment:
That reads very strange, 18 quitillion tuples on 8kB. Can you copy/paste the pgbench example, or maybe use the pagila example? (it would be fair to use the output of `make tests/pagila` here).

Reply:
I don't see it now. I will update this and when we encounter this case once more we can open an issue for it.